Tests/AttributedStringStyleTests.swift

```diff
@@ -37,7 +37,7 @@ class StringStyleTests: XCTestCase {
     #if os(iOS) || os(tvOS)
     func testTextStyle() {
         let style = StringStyle(.textStyle(titleTextStyle))
-        for (style, fullStyle) in additiviePermutations(for: style) {
+        for (style, fullStyle) in additivePermutations(for: style) {
```

Comment:
while we're here, why don't we prefer forEach instead? 
```suggestion 
       additivePermutations(for: style).forEach { (style, fullStyle) in
```